components/docs-chef-io/content/automate/ha_aws_deploy_steps.md

```diff
@@ -110,7 +110,7 @@ Run the following steps on Bastion Host Machine:
     - If you don't have a custom FQDN leave `fqdn` as empty for this AWS deployment. By default, AWS Application load balancer will be used as `fqdn`.
     - Set `instance_count` for Chef Automate, Chef Infra Server, Postgresql, OpenSearch.
     - Set AWS Config Details:
-        - Set `profile`, by default `profile` is `"default"`
+        - Set `profile`, which has your local AWS Credentials, if not given it will pick IAM Role attached to Bastion.
```

Comment:
```suggestion
        - Set the `profile`. Incase you attached IAM role to the bastion machine, leave this field empty
```

Reply:
Updated the statement

Reply:
Better yet make it : "Leave this field empty if you have an IAM role attached to the bastion machine"